frontend/src/components/ToDoLists.js

```diff
@@ -64,6 +64,16 @@ export const ToDoLists = ({ userState }) => {
             setSelectedList(res)
         }
     }
+    // handler for deleting list
+    const handleDeleteList = async (listId) => {
+        if (!window.confirm("Are you sure?")) {
+            return;
+        }
+        await deleteList(listId);
+        const newLists = lists.filter(l => l.id !== listId ? l : null)
+        setLists(newLists)
+        setSelectedList(newLists[0])
```

Comment:
This would reset the selected list to the first list in lists, what happens when we have 3 lists, the third is selected, and we delete the second?